### ISSUE TITLE
Releases / status quo?

### ISSUE THREAD

**Issue by Datseris** [tkf/LyapunovExponents.jl#7]
Hey

I wanted to link this repo to the docstring of the lyapunov function in ChaosTools. But checking the releases it looks like there aren't any.

What is the status quo? Are you still working on this repo? If yes, do you mind tagging stable releases? Also what is the plan for updating to julia 0.7?

**Comment by tkf** [comment]
Hi, sorry, I notice this issue just now.  There is no planned release at the moment.  I want to add a few more features before solidifying the interface (though releasing as is may not be so bad).  I was planning to migrate to Julia 0.7 after all the dependencies have the support.  DiffEq and Plots are the big ones and they were not ready last time I checked.

I can send an issue in ChaosTools once it's released.  Does it sound fine?

**Comment by Datseris** [comment]
Hey, no problem.

> I can send an issue in ChaosTools once it's released. Does it sound fine?

Sure please do. Since I already started updating to 0.7 (and only the ODE part is now left) please do it after you have also brought the repo to to 0.7. 

By the way, how do you get the MLE here? Do you create some sort of parallel integrator as well or you get the first entry of the QR result?

**Comment by tkf** [comment]
> do it after you have also brought the repo to to 0.7

Sure.

> how do you get the MLE here?

I have a separate solver for MLE though you can set the desired dimension to one for the QR-based method to do the same. The MLE solver just normalizes the associated tangent vector repeatedly. I think it's the same as how you do it in ChaosTools.

**Comment by Datseris** [comment]
>  I think it's the same as how you do it in ChaosTools.

Nope, it is very different. So in principal here you are using the identical method with multiple exponents but you use one deviation vector instead of many.

In ChaosTools we have a setting where two nearby states are evolved in time simultaneously and their distance is recorded and rescaled constantly. There is no use of the Jacobian of the equations of motion. 

This is good, because this means that two different algorithms exist in Julia ;)
